libs/langchain/pyproject.toml

```diff
@@ -147,6 +147,8 @@ praw = {version = "^7.7.1", optional = true}
 msal = {version = "^1.25.0", optional = true}
 databricks-vectorsearch = {version = "^0.21", optional = true}
 dgml-utils = {version = "^0.3.0", optional = true}
+cinemagoer = {git = "https://github.com/cinemagoer/cinemagoer", optional = true}
```

Comment:
seems like there's a published version of the package, any reason not to use that? https://pypi.org/project/cinemagoer/

Reply:
I believe we had issues getting the published version of Cinemagoer to work reliably, which went away once we started installing the package directly from the repository. Installing from the repo does seem to be the preferred installation method, as indicated in the documentation: https://cinemagoer.readthedocs.io/en/latest/#installation